Rakefile

```diff
@@ -4,3 +4,8 @@
 require File.expand_path("../config/application", __FILE__)
 
 AppPrototype::Application.load_tasks
+
+if Rails.env.development? || Rails.env.test?
+  Rake::Task[:default].clear_prerequisites
+  task default: %i[spec lint]
+end
```

Comment:
I felt it was more discoverable to define the default rake task here, in the root Rakefile. Before it was hiding in `lib/tasks/rubocop.rake`.